<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Label Propagation Framework</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,15" columnstretch="1,15">
        <item row="0" column="0" rowspan="2">
         <widget class="QFrame" name="frame_9">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0,0,0,0">
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="minimumSize">
              <size>
               <width>357</width>
               <height>41</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>75</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_openFile">
                <property name="text">
                 <string>Open File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="GraphList">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>45</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_25">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QGroupBox" name="groupBox_9">
                <property name="title">
                 <string>Setup</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Algorithm</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
                    <item>
                     <widget class="QRadioButton" name="rb_sync">
                      <property name="text">
                       <string>Sync</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_async">
                      <property name="text">
                       <string>Async</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_5">
                   <property name="title">
                    <string>Init Labels</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_2" columnstretch="6,0,0,0">
                    <item row="0" column="2">
                     <widget class="QLabel" name="label_13">
                      <property name="text">
                       <string>K</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="3">
                     <widget class="QSpinBox" name="sb_K_2">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>2</number>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QRadioButton" name="rb_Random_2">
                      <property name="mouseTracking">
                       <bool>false</bool>
                      </property>
                      <property name="toolTip">
                       <string>Random</string>
                      </property>
                      <property name="text">
                       <string>Rand</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QRadioButton" name="rb_Unique_2">
                      <property name="mouseTracking">
                       <bool>false</bool>
                      </property>
                      <property name="focusPolicy">
                       <enum>Qt::ClickFocus</enum>
                      </property>
                      <property name="toolTip">
                       <string>huhu</string>
                      </property>
                      <property name="whatsThis">
                       <string extracomment="SchÃ¶n"/>
                      </property>
                      <property name="text">
                       <string>Unique</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_2">
                   <property name="title">
                    <string>Update Order</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <item>
                     <widget class="QRadioButton" name="rb_Fixed">
                      <property name="text">
                       <string>Fixed Order</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_Rnd">
                      <property name="text">
                       <string>Random Order</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="title">
                    <string>Neighbors</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0,0">
                    <item row="1" column="3">
                     <widget class="QRadioButton" name="rb_Hop2">
                      <property name="text">
                       <string>Hop 2</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QRadioButton" name="rb_Hop0">
                      <property name="text">
                       <string>Hop 0 - 1</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QRadioButton" name="rb_Hop1">
                      <property name="text">
                       <string>Hop 1</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="4">
                     <widget class="QRadioButton" name="rb_Hop02">
                      <property name="text">
                       <string>Hop 0 - 2</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_11">
                   <property name="title">
                    <string>Label Selection</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1,0,0,0">
                    <item>
                     <widget class="QRadioButton" name="rb_RLabel">
                      <property name="text">
                       <string>Random</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_SLabel">
                      <property name="text">
                       <string>Self</string>
                      </property>
                      <property name="checked">
                       <bool>false</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_first">
                      <property name="text">
                       <string>First</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_Last">
                      <property name="text">
                       <string>Last</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_Median">
                      <property name="text">
                       <string>Median</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_12">
                   <property name="title">
                    <string>Convergence</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0,0">
                    <item>
                     <widget class="QRadioButton" name="rb_Stabil">
                      <property name="text">
                       <string>Stabil</string>
                      </property>
                      <property name="checked">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QRadioButton" name="rb_MaxIter">
                      <property name="text">
                       <string>Max Iter</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QSpinBox" name="sb_MaxIter">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="minimum">
                       <number>5</number>
                      </property>
                      <property name="maximum">
                       <number>100</number>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QFrame" name="frame_5">
                   <property name="frameShape">
                    <enum>QFrame::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Raised</enum>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_9">
                    <item>
                     <widget class="QPushButton" name="btn_LPA">
                      <property name="text">
                       <string>Standart LPA</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="btn_RunAll">
                      <property name="text">
                       <string>Full Test</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout" columnstretch="4,0,0,0">
              <item row="0" column="2">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Number of Tests</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3" rowspan="2">
               <widget class="QSpinBox" name="sb_MaxTest">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>2000</number>
                </property>
                <property name="value">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="rb_visual">
                <property name="text">
                 <string>Visual Graph</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="rb_noVisual">
                <property name="text">
                 <string>No Visualisation</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="btn_run">
                <property name="text">
                 <string>Run</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Step">
                <property name="text">
                 <string>Step</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Pause">
                <property name="text">
                 <string>Pause</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_Stop">
                <property name="text">
                 <string>Stop</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="btn_save">
                <property name="text">
                 <string>Save Graph</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_saveTable">
                <property name="text">
                 <string>Save Table</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Seite</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="font">
           <font>
            <family>Arial</family>
           </font>
          </property>
          <column>
           <property name="text">
            <string>Graph</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mode</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Initialize</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>K</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Update Order</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Neighbors</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Label Sel.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Convergence Rule</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Modularity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Best Modularity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Convergence</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Iterations</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Compare with</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Modularity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
